src/helpers/get-insights-anonymous-user-token.ts

```diff
@@ -0,0 +1,21 @@
+export const ANONYMOUS_TOKEN_COOKIE_KEY = '_ALGOLIA';
```

Comment:
this is exported just for the test right?

Reply:
yes